content/spin/redis-trigger.md

```diff
@@ -1,20 +1,27 @@
 title = "The Spin Redis trigger"
 template = "spin_main"
 date = "2022-03-14T00:22:56Z"
+enable_shortcodes = true
 [extra]
 url = "https://github.com/fermyon/spin/blob/main/docs/content/redis-trigger.md"
 
 ---
-- [The WebAssembly Interface](#the-webassembly-interface)
+- [Specifying an Application as Redis](#specifying-an-application-as-redis)
+- [Mapping a Channel to a Component](#mapping-a-channel-to-a-component)
+- [Redis Components](#redis-components)
+  - [The Message Handler](#the-message-handler)
+- [Inside Redis Components](#inside-redis-components)
 
-Spin applications can be triggered by a new message on a [Redis channel](https://redis.io/topics/pubsub).
-Spin will connect to a configured Redis instance and will invoke components for
-new messages on the configured channels.
+Pub-sub-style messaging is a popular architecture for asynchronous message processing. Spin has built-in support to creating and running applications in response to messages on [pub-sub Redis channels](https://redis.io/topics/pubsub).
```

Comment:
Nit: my attention was strangely drawn to `pub-sub-style` (possibly due to caffeine deficiency); maybe just `pub-sub messaging`?

Reply:
Good call, also expanded the abbreviation.